gatsby-theme-patternfly-org/helpers/codesandbox.js

```diff
@@ -41,6 +43,13 @@ export const getReactParams = (title, code) => {
     toRender = equalityMatch[1];
     code = code.replace(/(\w+) =/, `const ${toRender} =`)
   }
+
+  const dependencies = {};
```

Comment:
Nitpick: you can just put `react: '^16.8.0', react-dom: '^16.8.0'` in here to begin with.